config/initializers/redis.rb

```diff
@@ -1,5 +1,6 @@
+port = Toxiproxy.running? ? 22220 : 6379
```

Comment:
Consider exposing this from the initializer so you don't have to hardcode the port twice.